admin-console/src/test/java/com/agiletec/plugins/jacms/apsadmin/content/TestContentAdminAction.java

```diff
@@ -39,6 +39,9 @@ protected void setUp() throws Exception {
 
     public void testOpenIndexProspect() throws Throwable {
         String result = this.executeOpenIndexProspect("admin");
+        synchronized (this) {
```

Comment:
Don't like this ... What is going on that this is required?

Reply:
reverted wrong commit